common/properties/coverageLevels.txt

```diff
@@ -0,0 +1,2 @@
+#Data file	providing the Coverage Level of locales at Basic or Above.#For more info,	see the Locale Coverage Chart for this version.#Locale ;	Level
+af ;	modernam ;	modernar ;	modernas ;	modernast ;	basicaz ;	modernbe ;	modernbg ;	modernbn ;	modernbr ;	moderatebrx ;	basicbs ;	modernbs_Cyrl ;	basicca ;	modernceb ;	moderatechr ;	moderncs ;	moderncy ;	modernda ;	modernde ;	moderndoi ;	basicdsb ;	modernel ;	modernen ;	modernes ;	modernet ;	moderneu ;	modernfa ;	modernff_Adlm ;	moderatefi ;	modernfil ;	modernfo ;	moderatefr ;	modernga ;	moderngd ;	moderngl ;	moderngu ;	modernha ;	modernhe ;	modernhi ;	modernhi_Latn ;	modernhr ;	modernhsb ;	modernhu ;	modernhy ;	modernia ;	basicid ;	modernig ;	moderateis ;	modernit ;	modernja ;	modernjv ;	modernka ;	modernkea ;	basickgp ;	moderatekk ;	modernkm ;	modernkn ;	modernko ;	modernkok ;	modernks ;	basicks_Deva ;	basicky ;	modernlo ;	modernlt ;	modernlv ;	modernmai ;	basicmi ;	basicmk ;	modernml ;	modernmn ;	modernmni ;	basicmr ;	modernms ;	modernmy ;	modernne ;	modernnl ;	modernnn ;	modernno ;	modernor ;	modernpa ;	modernpcm ;	modernpl ;	modernps ;	modernpt ;	modernqu ;	moderaterm ;	basicro ;	modernru ;	modernsa ;	basicsat ;	basicsc ;	moderatesd ;	moderatesd_Deva ;	basicsi ;	modernsk ;	modernsl ;	modernso ;	modernsq ;	moderatesr ;	modernsu ;	basicsv ;	modernsw ;	modernta ;	modernte ;	moderntg ;	basicth ;	moderntk ;	modernto ;	basictr ;	moderntt ;	basicuk ;	modernur ;	modernuz ;	modernuz_Cyrl ;	basicvi ;	modernwo ;	basicyo ;	moderateyrl ;	moderateyue ;	modernyue_Hans ;	basiczh ;	modernzh_Hant ;	modernzu ;	moderate#EOF	
```

Comment:
Looks wrong, new lines missing?

Reply:
whoops! Did some code cleanup at the end, and missed this.